ChatSecure/Classes/View Controllers/OTRMessagesViewController.m

```diff
@@ -1227,6 +1253,10 @@ - (void)didFinishTyping {
 
 #pragma - mark Sending Media Items
 
+- (BOOL)canSendMedia {
+    return self.state.canSendMedia || [self isGroupChat];
```

Comment:
This isn't entirely accurate. I think it should just be `self.state.canSendMedia`, because otherwise you will think you can send media in group chats even if you can't.